loadtest/user/userentity/helper_test.go

```diff
@@ -46,10 +46,11 @@ type config struct {
 		PercentGroupChannels        float64 `default:"0.1" validate:"range:[0,1]"`
 	}
 	UsersConfiguration struct {
```

Comment:
Just curious why do we have these structs duplicated for this test?

Reply:
There was an import loop somewhere or some other problem due to which I had to duplicate it.